packages/now-build-utils/src/types.ts

```diff
@@ -99,3 +99,26 @@ export interface PrepareCacheOptions {
    */
   config: Config;
 }
+
+export interface ShouldServeParams {
```

Comment:
```suggestion
export interface ShouldServeOptions {
```

Can we make this match the others?

Reply:
Thought about that, but I felt it's more like **params** than **options** to  `shouldServe`. "options" might more suits to some function with side effect. 🤔

Reply:
I made a bad decision when I added TypeSCript and used `AnalyzeOptions` `BuildOptions`, and `PrepareCacheOptions` to name these inputs. These are already documented here: https://zeit.co/docs/v2/deployments/builders/developer-guide/#builder-exports

I think we need to maintain the same naming convention when introducing to params/options.